spark/src/main/scala/org/apache/comet/serde/bitwise.scala

```diff
@@ -0,0 +1,161 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.comet.serde
+
+import org.apache.spark.sql.catalyst.expressions._
+import org.apache.spark.sql.types.{ByteType, IntegerType, LongType}
+
+import org.apache.comet.serde.QueryPlanSerde._
+
+object CometBitwiseAdd extends CometExpressionSerde {
+  override def convert(
+      expr: Expression,
+      inputs: Seq[Attribute],
+      binding: Boolean): Option[ExprOuterClass.Expr] = {
+    val bitwiseAndExpr = expr.asInstanceOf[BitwiseAnd]
+    createBinaryExpr(
+      expr,
+      bitwiseAndExpr.left,
+      bitwiseAndExpr.right,
+      inputs,
+      binding,
+      (builder, binaryExpr) => builder.setBitwiseAnd(binaryExpr))
+  }
+}
+
+object CometBitwiseNot extends CometExpressionSerde {
+  override def convert(
+      expr: Expression,
+      inputs: Seq[Attribute],
+      binding: Boolean): Option[ExprOuterClass.Expr] = {
+    val bitwiseNotExpr = expr.asInstanceOf[BitwiseNot]
+    val childProto = exprToProto(bitwiseNotExpr.child, inputs, binding)
+    val bitNotScalarExpr =
+      scalarFunctionExprToProto("bit_not", childProto)
```

Comment:
I can't find such function 🤔

Reply:
This function based on: native/spark-expr/src/bitwise_funcs/bitwise_not.rs